ocrd/ocrd/processor/base.py

```diff
@@ -15,9 +15,11 @@
 
 class Processor():
     """
-    A processor runs an algorithm based on the workspace, the mets.xml in the
-    workspace (and the input files defined therein) as well as optional
-    parameter.
+    A processor is an OCR-D compliant command-line-interface for executing
```

Comment:
@EEngl52 @cneud perhaps update our glossary entry on *processor* https://ocr-d.de/en/spec/glossary#processor?

Reply:
For me this definition is way more understandable than the one currently on our website

Reply:
https://github.com/OCR-D/ocrd-website/pull/183